tests/integration-tests/test_runner.py

```diff
@@ -402,6 +403,13 @@ def _init_argparser():
         default=TEST_DEFAULTS.get("external_shared_storage_stack_name"),
     )
 
+    debug_group.add_argument(
+        "--force-run-instance-fleet-manager",
+        help="Forces the usage of EC2 run-instances API for creation of Fleet",
+        default=TEST_DEFAULTS.get("force_run_instance_fleet_manager"),
```

Comment:
the boto3 api is "run-instances" not run-instance, this is why I suggest to rename it.